.github/workflows/mo.yml

```diff
@@ -52,6 +52,10 @@ jobs:
           pip install -r requirements_dev.txt
         working-directory: tools/mo
 
-      - name: Pylint
+      - name: Pylint-MO
         run: pylint -d C,R,W openvino/tools/mo
         working-directory: tools/mo
+
+      - name: Pylint-OVC
+        run: pylint -d C,R,W openvino/tools/ovc
+        working-directory: tools/ovc
```

Comment:
can you please double-check this one because it seems to generate empty report for some reason?

Reply:
Yes, I missed the config files for pylint, so the check was not complete. Added same configs as in MO.